docs/help.md

```diff
@@ -1,13 +1,13 @@
 `apimetrics help`
 =================
 
-Display help for apimetrics.
+Display help for APImetrics CLI.
```

Comment:
This is automatically generated. The reason it is apimetrics over APImetrics CLI is because apimetrics is the name of command